tests/digraph/test_spring_layout.py

```diff
@@ -0,0 +1,64 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import unittest
+
+import retworkx
+
+
+class TestSpringLayout(unittest.TestCase):
+
```

Comment:
It might be better to add tests which examine the accuracy of output positions, such as comparing to outputs generated by networkx’s spring layout function

Reply:
Well, i'm not sure. This function and networkx's spring layout function don't give identical results but rather "equivalent" since:
- there are some hardcoded values that may not match
- there are some parameters that networkx doesn't expose (e.g repulsive force exponent, cooling scheme)
- we do a kind of numeric integration that's quite sensitive in the order you process nodes, "when" you apply forces etc.